app/assets/scripts/map.js

```diff
@@ -480,17 +495,18 @@ function populateMarkers(resources) {
   for (var i = 0; i < resources.length; i++) {
     createMarker(resources[i]);
   }
-
   var bounds = new google.maps.LatLngBounds();
   for (var i = 0; i < markers.length; i++) {
     markers[i].setMap(map);
     oms.addMarker(markers[i]);
     bounds.extend(markers[i].getPosition());
   }
-
+  allResourceBounds = bounds;
   map.fitBounds(bounds);
   map.setCenter(bounds.getCenter());
-  allResourceBounds = bounds;
+  if (markers.length == 1) {
+    map.setZoom(15);
```

Comment:
this is so small but: thoughts on zooming this out a little farther? the super close zoom was a bit disorienting for me.

Reply:
On it!

Reply:
Actually this zoom level is good, there's some other weird issue right now, I'll look into it